### DISCUSSION TITLE
请求增加图像编辑中箭头、记号笔等标注工具

### DISCUSSION THREAD

**Discussion by wu-ricky** [xushengfeng/eSearch#118] [Ideas]
截图最常用其实还是图片标注功能，比如箭头（多样式）、记号笔、箭头文本……如果可以增加窗口识别就更方便，加油！！！
![Snipaste_2023-01-14_02-33-56](https://user-images.githubusercontent.com/28890827/212394106-e53ce550-ca64-4ee6-ab68-e3739f4c1b98.png)
![eSearch-2023-01-14-02-36-40-528](https://user-images.githubusercontent.com/28890827/212394376-c54048d0-cb59-45f0-af15-e5c0e27d790b.png)

**Comment by xushengfeng** [comment]
箭头更多样式考虑添加
记号笔具体是怎样的？如果是那种粗线半透明的话，可以在颜色那里调
像箭头文本这样的复杂图像应会推出自定义功能的

**Reply by yintianhuaxiaohao1** [reply]
6666
